aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -106,6 +107,18 @@ def physical_pixel_sizes(self) -> PhysicalPixelSizes:
             self._physical_pixel_sizes = PhysicalPixelSizes(z_size, y_size, x_size)
         return self._physical_pixel_sizes
 
+    @property
```

Comment:
Normally, channel names can change per-scene in other formats. I would assume it is the same for Zarr but if not, ignore this!

Reply:
My understanding is in OME-Zarr "scenes" are just broken down into separate zarrs where each one can get its own .zattrs metadata with its own omero channel names.  We have not yet adopted or supported the HCS spec which deals with plates/wells.  So for this code, multiscene doesn't mean much, it's pretty much always scene 0 i think?